src/sdk/modules/utils/Helpers.ts

```diff
@@ -102,7 +102,7 @@ export const parseModule = <
   chain extends Chain | undefined
 >(
   client: Client<Transport, chain, TModularSmartAccount>
-) => {
+): AnyData => {
```

Comment:
what does parseModule do?

Reply:
It's a helper meant only for internal use. Not to be used externally really. It's a helper that pulls out the active module instance from the nexusClient.